eq-author-api/schema/tests/section.test.js

```diff
@@ -212,5 +212,86 @@ describe("section", () => {
       });
       expect(queriedSection.validationErrorInfo.errors).toHaveLength(1);
     });
+
+    describe("section page", () => {
+      let section;
+
+      beforeEach(async () => {
+        ctx = await buildContext({
+          sections: [
+            {
+              title: "",
+            },
+          ],
+        });
+
+        questionnaire = ctx.questionnaire;
+        section = questionnaire.sections[0];
+      });
+
+      it("should validate if neither introduction title nor introduction content are populated", async () => {
```

Comment:
Should this be "should be valid if..."? Same for line 248?

Reply:
Test comments changed to be clearer.